components/brave_wallet_ui/components/extension/confirm-transaction-panel/index.tsx

```diff
@@ -144,11 +157,44 @@ function ConfirmTransactionPanel (props: Props) {
   const [currentTokenAllowance, setCurrentTokenAllowance] = React.useState<string>('')
   const [isEditingAllowance, setIsEditingAllowance] = React.useState<boolean>(false)
   const [showAdvancedTransactionSettings, setShowAdvancedTransactionSettings] = React.useState<boolean>(false)
+  const [showTransactionsBackedUpWarning, setShowTransactionsBackedUpWarning] = React.useState<boolean>(false)
 
   const { findAssetPrice } = usePricing(transactionSpotPrices)
   const parseTransaction = useTransactionParser(selectedNetwork, accounts, transactionSpotPrices, visibleTokens, fullTokenList)
   const transactionDetails = parseTransaction(transactionInfo)
 
+  const calculateDiffInMilliseconds = (date: Date) => {
```

Comment:
This can be moved outside of the component

Reply:
Good point!
Moved to our `datetime-utils` where it is also being used in our `tests` now.